lab-alex-bessie-david/lib/transformer.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const transformer = module.exports = function(buff, transformType){
+  if(!Buffer.isBuffer(buff)) throw new Error('not a buffer');
+  var transformTypes = ['blackout', 'invert', 'grayscale', 'invcolors'];
+  if(!transformTypes.includes(transformType)) throw new Error('transform type not supported');
+  
+  switch(transformType){
```

Comment:
You have a lot of repeated actions in this switch statement. For example, finding the start and offset. It would be much cleaner if you had a function that would do that for you and return the values that you need. Even nicer, if each transform were its own function, your switch case would be lean and clean. User wants this transform, switch case jumps there and calls that function.